src/platform/packages/shared/kbn-es/src/serverless_resources/project_roles/security/roles.yml

```diff
@@ -1,6 +1,5 @@
 # -----
-# Copy from internal roles config in elasticsearch-controller
-# -----
+# Source: https://github.com/elastic/elasticsearch-controller/blob/main/internal/config/roles/security.yaml# -----
```

Comment:
Please remove the full path to controller, we don't do it on purpose (public vs private repo)

Reply:
done

Reply:
also added TODOs 855cf2b51c34ec1ca7a77fabff3d66b3bb051bec